sqitch.plan

```diff
@@ -91,3 +91,5 @@ swrs/transform/function/load_fuel_mapping [swrs/transform/function/load_fuel_map
 @v1.0.1 2020-06-30T22:52:10Z Dylan Leard <dylan@button.is> # Version 1.0.1
 swrs/transform/function/load_emission [swrs/transform/function/load_emission@v1.0.1] 2020-09-25T18:54:54Z Dylan Leard <dylan@button.is> # Migration: add missing fuel_mapping foreign key
 @v1.0.2 2020-10-15T18:35:55Z Matthieu Foucault <matthieu@button.is> # Version 1.0.2
+
```

Comment:
Not sure if this empty line could be an issue in the future, but that's unusual. Maybe remove it?

Reply:
I removed it. Maybe there were 2 empty lines at the end of the file before I made my changes?